pkg/metrics/metrics.go

```diff
@@ -11,10 +11,17 @@ import (
 )
 
 const (
-	MetricNamespace = "nsx"
-	MetricSubsystem = "operator"
-	HealthKey       = "health_status"
-	ScrapeTimeout   = 30
+	MetricNamespace                 = "nsx"
+	MetricSubsystem                 = "operator"
+	HealthKey                       = "health_status"
+	ControllerSyncTotalKey          = "controller_sync_total"
```

Comment:
Later if we support more controllers, do we need to define other metrics for sync/update/delete? Or we reuse these ones.

Reply:
we can use these ones with different res_type